example/src/Root.js

```diff
@@ -19,6 +19,8 @@ stripe.setOptions({
   androidPayMode: 'test',
 })
 
+stripe.setStripeAccount('<STRIPE_ACCOUNT>')
```

Comment:
Can you change the above line to this instead?  This will preserve existing behaviour for non-connect accounts (where <STRIPE_ACCOUNT> is not defined)
```
const connectedAccount = '<STRIPE_ACCOUNT>';
if (connectedAccount != '') {
  stripe.setStripeAccount(connectedAccount)
}
```

Reply:
Done